CHANGELOG.md

```diff
@@ -5,10 +5,12 @@ Full documentation for rocThrust is available at [https://rocthrust.readthedocs.
 ## (Unreleased) rocThrust 2.18.0 for ROCm 5.6
```

Comment:
This will go in for ROCm 5.7. I'll update the changelog and the package version in CMake.

Reply:
Ah, the PR is coming from the Stream repo.  Can you update the changelog entry to 5.7?

Reply:
@nolmoonen @vince-streamhpc @Snektron Hi, can one of you update the changelog entry to indicate ROCm 5.7? Thanks.  I'd like to get this merged in before Wednesday.

Reply:
I changed it. Sorry for the delay

Reply:
No worries. Thanks!